osclib/stagingapi.py

```diff
@@ -1150,10 +1150,13 @@ def submit_to_prj(self, act, project):
 
         # If adi project, check for baselibs.conf in all specs to catch both
         # dynamically generated and static baselibs.conf.
-        baselibs = False if self.is_adi_project(project) else None
-        if baselibs is False and 'baselibs.conf' in source_file_load(
-                self.apiurl, src_prj, src_pkg, '{}.spec'.format(src_pkg), src_rev):
-            baselibs = True
+        if self.is_adi_project(project):
+            baselibs = False
+            specfile = source_file_load(self.apiurl, src_prj, src_pkg, '{}.spec'.format(src_pkg), src_rev)
+            if specfile and 'baselibs.conf' in specfile:
+               baselibs = True
```

Comment:
./osclib/stagingapi.py:1157:16: E111 indentation is not a multiple of four

Reply:
I prefer prime numbers - always did :)